mathesar/api/serializers/tables.py

```diff
@@ -1,12 +1,17 @@
 from django.urls import reverse
-from rest_framework import serializers
-from rest_framework.exceptions import ValidationError
+from psycopg2.errors import DuplicateTable
+from rest_framework import serializers, status
+from sqlalchemy.exc import ProgrammingError
 
+from mathesar.api.exceptions.exceptions import ProgrammingAPIException, DuplicateTableAPIException, MultipleDataFileAPIException, \
+    DistinctColumnRequiredAPIException, ColumnSizeMismatchAPIException
```

Comment:
I think either splitting up these imports or (even better) using parentheses would be better.  Escaped line breaks aren't usually needed in Python.